examples/integration/CommandLine/Tests/BUILD

```diff
@@ -2,23 +2,25 @@ load("@build_bazel_rules_apple//apple:macos.bzl", "macos_unit_test")
 load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
 
 swift_library(
-    name = "LibSwiftTestsLib",
+    name = "CommandLineToolTests",
```

Comment:
Sorry, meant to have you rename the other one. This can be `CommandLineToolTestsLib` and below can be `CommandLineToolTests`.